### PR TITLE
Indicate if the Helm chart has a schema

### PR DESCRIPTION
**Pull request by nwmac**
- Refine the chart view
- Indicate if the chart has a schema available
- Show the download link for the chart
- Fixes issue where side panel grew too bug for long home page URLs
- Only enables the install button if a Kubernetes endpoint is connected

### CODE REVIEW

File: src/frontend/packages/suse-extensions/src/custom/helm/monocular/chart-details/chart-details-info/chart-details-info.component.ts [hunk 1]
```diff
@@ -12,8 +14,22 @@ import { ChartsService } from '../../shared/services/charts.service';
 })
 export class ChartDetailsInfoComponent implements OnInit {
   @Input() chart: Chart;
-  @Input() currentVersion: ChartVersion;
   versions: ChartVersion[];
+  schema: any = null;
+
+  _currentVersion: ChartVersion;
```
Comment (nwmac) [inline src/frontend/packages/suse-extensions/src/custom/helm/monocular/chart-details/chart-details-info/chart-details-info.component.ts:20]: pCurrentVersion

File: src/frontend/packages/suse-extensions/src/custom/helm/monocular/chart-details/chart-details-usage/chart-details-usage.component.html [hunk 2]
```diff
@@ -1,4 +1,11 @@
 <div class="chart-details-usage__install">
-  <button class="chart-details-usage__install-btn" color="primary" mat-button mat-raised-button
+  <button *ngIf="endpointsService.hasConnectedEndpointType('k8s') | async; else nok8s" class="chart-details-usage__install-btn" color="primary" mat-button mat-raised-button
   routerLink="/monocular/install/{{chart.id}}/{{currentVersion}}">Install Chart</button>
 </div>
+
+<!-- If there are no Kubernetes endpoints connected, show a disabled buitton with a tool tip-->
+<ng-template #nok8s>
+  <div matTooltip="Add a Kubernetes endopoint to install charts">
```
Comment (nwmac) [inline src/frontend/packages/suse-extensions/src/custom/helm/monocular/chart-details/chart-details-usage/chart-details-usage.component.html]: typo

File: src/frontend/packages/suse-extensions/src/custom/helm/monocular/shared/services/charts.service.ts [hunk 3]
```diff
@@ -109,6 +109,19 @@ export class ChartsService {
       responseType: 'text'
     });
   }
+
+  /**
+   * Get a chart Values.json using the API
```
Comment (nwmac) [inline src/frontend/packages/suse-extensions/src/custom/helm/monocular/shared/services/charts.service.ts]: Update comment

**Review by richard-cox — APPROVED** [review]
LGTM

**Pull request merged**